crates/cargo-contract/src/cmd/config.rs

```diff
@@ -143,73 +138,75 @@ impl SignerConfig<Self> for Polkadot {
 
 /// Struct representing the implementation of the sr25519 signer
 #[derive(Clone)]
-pub struct SignerSR25519<C: Config>(pub PairSigner<C, sp_core::sr25519::Pair>);
+pub struct SignerSR25519<C: Config> {
+    account_id: <C as Config>::AccountId,
```

Comment:
PairSigner no longer available, guidance from subxt release notes is to follow https://github.com/paritytech/subxt/blob/d924ece39a5cb369ba5ccde3dc160b5ee006271b/subxt/examples/substrate_compat_signer.rs